src/infrastructure/server/graphql/spot/resolver.ts

```diff
@@ -64,6 +66,7 @@ const Spot: SpotResolver = {
   },
   comments: (spot, _args, context) =>
     context.repositories.comment.findBySpotId(spot.id),
+  tags: spot => spot.tags.map(tag => `#${tag}`),
```

Comment:
I don't think we should consider tags as hashtags. This is more restrictive (e.g. no non-alphanumerical characters). We can stick to simple tags like [GitHub topics](https://github.com/topics).